tests/datalab/issue_manager/test_imbalance.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pytest
+
+from cleanlab.datalab.issue_manager.imbalance import ClassImbalanceIssueManager
+
+SEED = 42
+
+
+class TestClassImbalanceIssueManager:
+    @pytest.fixture
+    def embeddings(self, lab):
+        embeddings_array = np.arange(lab.get_info("statistics")["num_examples"] * 10).reshape(-1, 1)
+        return embeddings_array
+
+    @pytest.fixture
+    def labels(self, lab):
+        K = lab.get_info("statistics")["num_classes"]
+        N = lab.get_info("statistics")["num_examples"] * 10
+        labels = np.random.choice(np.arange(K - 1), size=N, p=[0.5] * (K - 1))
+        labels[0] = K - 1  # Rare class
+        return labels
+
+    @pytest.fixture
+    def issue_manager(self, lab, labels, monkeypatch):
+        monkeypatch.setattr(lab._labels, "labels", labels)
+        return ClassImbalanceIssueManager(datalab=lab, fraction=0.1)
+
+    def test_find_issues(self, embeddings, issue_manager):
+        issue_manager.find_issues(features=embeddings)
+        issues, summary = issue_manager.issues, issue_manager.summary
+        expected_issue_mask = np.array([True] * 1 + [False] * 49)
+        assert np.all(
+            issues["is_class_imbalance_issue"] == expected_issue_mask
+        ), "Issue mask should be correct"
+        assert summary["issue_type"][0] == "class_imbalance"
+        assert summary["score"][0] == pytest.approx(expected=0.9804, abs=1e-7)
```

Comment:
I'm thinking of two similar tests for `find_issues`?

1. What if there is no rare class. I expect that the issue_mask will be all falses. Will the scores just be ones? I think the test case will look similar to `test_find_issues`
2. What if there is more than one "rare" classe? Let's make `K-1` still the _most_ rare class, but what do you suggest we do about the "other rare class" in the test suite for now? Suppose `K-2` is the second rarest class, then we could check that it's not flagged as having an issue and it's score is still set to 1?

Reply:
1. That makes sense, yes all the scores will be one. 
2. I am assuming that in the presence of another rare class with `K-1` still being the rarest class, tests for both issue_mask and scores could be exactly same as the one in `test_find_issues`? Do we explicitly need to check for the `K-2` class having an issue, and it's score being 1?

Reply:
Re 2: I think we can just reuse the issue mask, but I'd like an explicit test that only _one_ class has an issue.

Reply:
I see. So something like `assert np.sum(issues["is_class_imbalance_issue"]) == 1` should do?

Reply:
Yeah, in fact, you can do this in the existing test as well!